transactions/transactions.py

```diff
@@ -36,6 +49,92 @@ async def genericAnnounce(self, ctx, *, message):
         except KeyError:
             await ctx.send(":x: Transaction log channel not set")
 
+    @commands.command(aliases=['makeFA'])
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def expireContracts(self, ctx: commands.Context, *userList):
+        """Displays each member that can be found from the userList a Free Agent in their respective tier"""
+        empty = True
+        free_agents = 0
+        notFound = 0
+        message = ""
+        fa_role = self.team_manager_cog._find_role_by_name(ctx, "Free Agent")
+        league_role = self.team_manager_cog._find_role_by_name(ctx, "League")
+
+        roles_to_remove = [
+            self.team_manager_cog._find_role_by_name(ctx, "Draft Eligible"),
+            self.team_manager_cog._find_role_by_name(ctx, self.PERM_FA_ROLE),
+            self.team_manager_cog._find_role_by_name(ctx, "Former Player")
+        ]
+
+        trans_channel : discord.TextChannel = await self._trans_channel(ctx)
+
+        for user in userList:
```

Comment:
might add too much complexity for this pass, but in future maybe we can extract out the code in this block and run the processing for each user concurrently?

Reply:
Yeah we can definitely revisit this.